pkg/controller/unifiedpushserver/unifiedpushserver.go

```diff
@@ -315,6 +327,16 @@ func newUnifiedPushServerDeploymentConfig(cr *pushv1alpha1.UnifiedPushServer) (*
 									ContainerPort: 4180,
 								},
 							},
+							Resources: corev1.ResourceRequirements{
+								Limits: corev1.ResourceList{
+									"memory": resource.MustParse("2Gi"),
+									"cpu":    resource.MustParse("20m"),
+								},
+								Requests: corev1.ResourceList{
+									"memory": resource.MustParse("2Gi"),
```

Comment:
do we need this much memory for the oauth proxy?

Reply:
Yeah looks like we updated the values in the spreadhseet.  I'll make it 32/64mi